pom.xml

```diff
@@ -93,13 +93,13 @@
         <release.endpoint>https://s01.oss.sonatype.org/</release.endpoint>
 
         <!-- Kafka and it's dependencies MUST reflect what the Kafka version uses -->
-        <version.kafka>3.4.0</version.kafka>
-        <version.zookeeper>3.6.3</version.zookeeper>
+        <version.kafka>3.5.0</version.kafka>
+        <version.zookeeper>3.6.4</version.zookeeper>
```

Comment:
As I'm seeing this, are there plans yet for phasing out the ZK dependency altogether? I.e. only use and show KRaft mode in embedded cluster, docs, tutorial, etc?